notebooks/120-tensorflow-object-detection-to-openvino/120-tensorflow-object-detection-to-openvino.ipynb

```diff
@@ -10,9 +10,9 @@
     "\n",
```

Comment:
<p>Optionally, we can apply compression to FP16 model weights using&nbsp;<code>compress_to_fp16=True</code>&nbsp;option and integrate preprocessing using this approach. - please, remove</p><p>or more information about Model Optimizer  - please, remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>removed!</p>